database/src/main/scala/cromwell/database/migration/metadata/table/CallOutputSymbolTableMigration.scala

```diff
@@ -0,0 +1,49 @@
+package cromwell.database.migration.metadata.table
+
+import java.sql.{PreparedStatement, ResultSet}
+
+import cromwell.database.migration.metadata.MetadataStatementForCall
+import wdl4s.values._
+
+class CallOutputSymbolTableMigration extends SymbolTableMigration {
+  override protected def selectQuery: String = """
+     SELECT SYMBOL.SCOPE, SYMBOL.NAME, MaxExecution.IDX, IO, WDL_TYPE,
+         WDL_VALUE, WORKFLOW_EXECUTION_UUID, ExecutionId.EXECUTION_ID, REPORTABLE_RESULT,
+         MaxExecution.MaxAttempt
+       FROM SYMBOL
+         LEFT JOIN WORKFLOW_EXECUTION ON SYMBOL.WORKFLOW_EXECUTION_ID = WORKFLOW_EXECUTION.WORKFLOW_EXECUTION_ID
+         LEFT JOIN(
+                    SELECT CALL_FQN, IDX, WORKFLOW_EXECUTION_ID, MAX(EXECUTION.ATTEMPT) AS MaxAttempt
+                    FROM EXECUTION
+                    GROUP BY EXECUTION.CALL_FQN, EXECUTION.IDX, EXECUTION.WORKFLOW_EXECUTION_ID
+                  ) MaxExecution
```

Comment:
I wonder if it would be faster to `MAX` on the execution ID rather than the attempt?